src/parsing/astHelper.ts

```diff
@@ -171,6 +171,8 @@ function followPath(ast: IAST, path: string[]): IAST | null {
 /**
  * Get the value of the given attribute
  */
+function getAttribute(ast: IAST, attributeName: 'type'): SequenceType | null;
+function getAttribute(ast: IAST, attributeName: Omit<string, 'type'>): string | SequenceType | null;
```

Comment:
This omit is actually clever, sort of like a default input if not passed I guess?

Reply:
Yeah, pretty much.